src/main/java/com/teamacronymcoders/epos/feature/quiver/QuiverCapabilityProvider.java

```diff
@@ -0,0 +1,29 @@
+package com.teamacronymcoders.epos.feature.quiver;
+
+import com.hrznstudio.titanium.block.tile.inventory.PosInvHandler;
+import net.minecraft.util.Direction;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+import net.minecraftforge.common.util.LazyOptional;
+import net.minecraftforge.items.CapabilityItemHandler;
+import net.minecraftforge.items.IItemHandler;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class QuiverCapabilityProvider implements ICapabilityProvider {
+    private final LazyOptional<IItemHandler> optional;
+
+    QuiverCapabilityProvider(PosInvHandler handler) {
```

Comment:
This could easily be made into a generic PosInvHandler Provider and not need to be quiver specific.